library/VTKExtensions/Rendering/glsl/vtkF3DBitonicSortLocalSortCS.glsl

```diff
@@ -0,0 +1,35 @@
+#version 430
+
+//VTK::BitonicDefines::Dec
+
+layout(local_size_x = WorkgroupSize) in;
+layout(std430) buffer;
+
+layout(binding = 0) buffer Keys
+{
+  KeyType key[];
+};
+
+layout(binding = 1) buffer Values
+{
+  ValueType value[];
+};
+
+layout(location = 0) uniform int count;
+
+//VTK::BitonicFunctions::Dec
+
+void main()
+{
+  for (uint h = 1; h <= WorkgroupSize; h *= 2)
+  {
+    barrier();
+    compare_and_swap(flip(gl_GlobalInvocationID.x, h));
```

Comment:
we cant retrieve directly in `flip` (same for `disperse`) method the `gl_GlobalInvocationID.x` ?

Reply:
We can, I'll change that